ProcessLib/HydroMechanics/HydroMechanicsFEM.h

```diff
@@ -566,6 +582,20 @@ class HydroMechanicsLocalAssembler : public LocalAssemblerInterface
         return cache;
     }
 
+    void assembleWithJacobianForDeformationEquations(
```

Comment:
docs missing here and below for pressure eqs.

Reply:
Added.